TurkishFilter/sections/whitelist.txt

```diff
@@ -1,4 +1,6 @@
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/74572
+@@||mcdn01.gittigidiyor.net/rw/media/*$domain=gittigidiyor.com
```

Comment:
Hm, too wide rule.
1. You should add `~gittigidiyor.com` to the `/reklam-$domain=~avansas.com`.
2. Exchange your exception to `@@||mcdn*.gittigidiyor.net/*/media/images/*/ads-packages.css$domain=gittigidiyor.com`

Reply:
But `/reklam-$domain=~avansas.com` rule is in this general_url file, are you sure to add in this?

Reply:
Ofc, that rule is caused it.
After your changes about `1.` it should be `/reklam-$domain=~avansas.com|~gittigidiyor.com`

Reply:
`~` karakteri istisna demektir. Önüne eklenen alan adını beyaz listeye alır. @gork7777 

https://kb.adguard.com/en/general/how-to-create-your-own-ad-filters#basic-modifiers

Reply:
Don't worry, I made the necessary changes.